earth_enterprise/src/common/serverdb/serverdb.cpp

```diff
@@ -93,35 +93,35 @@ bool ServerdbConfig::Load(std::string config_file) {
 
 
 bool ServerdbConfig::Save(std::string config_file) {
-  FILE* fp = fopen(config_file.c_str(), "w");
-  if (fp == NULL) {
+  std::unique_ptr<AbstractFileAccessor> aFA = AbstractFileAccessor::getAccessor(config_file);
+  aFA->Open(config_file.c_str(), "w");
+  if (!aFA->isValid()) {
     notify(NFY_WARN, "Unable to save ServerDB config file %s",
            config_file.c_str());
     return false;
   }
-  khFILECloser closer(fp);
 
-  fprintf(fp, "%s%s%d\n", kDbType.c_str(), kDelim.c_str(), db_type);
+  fprintf(aFA->getFP(), "%s%s%d\n", kDbType.c_str(), kDelim.c_str(), db_type);
```

Comment:
The `fprintf` calls are directly modifying disk files, so we probably want to make an equivalent method in `FileAccessor` and call that instead.  That would probably let us get rid of the `getFP()` method entirely as well.